doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -263,13 +263,17 @@ The diagnostic logs snippet enables the set of configurations needed for full Ma
 The configuration set consist of devicetree overlays for each supported target board, and a config file that enables all diagnostic logs features by default.
 The devicetree overlays add new RAM partitions which are configured as retained to keep the log data persistent and survive the device reboot.
 They also reduce the SRAM size according to the size of the retained partition.
+The parition sizes are configured using example values and may not be sufficient for your use cases.
+To change the partition sizes, for example increase them to be able to store more logs, you need to change the configuration in the devicetree overlay.
```

Comment:
```suggestion
To change the partition sizes, you need to change the configuration in the devicetree overlay.
You can, for example, increase the partition sizes to be able to store more logs.

```
Note the empty line at the end. The paragraph is long enough now that it makes sense to separate the sentence introducing the Kconfig options from the rest of the text.